src/NHibernate.Test/Async/Ado/BatcherFixture.cs

```diff
@@ -102,44 +102,7 @@ public async Task OneRoundTripUpdateAsync()
 			await (CleanupAsync());
 		}
 
-#if NETFX
-		[Test, Ignore("Not fixed yet.")]
-		[Description("SqlClient: The batcher should run all different INSERT queries in only one roundtrip.")]
-		public async Task SqlClientOneRoundTripForUpdateAndInsertAsync()
```

Comment:
I didn't get it why this was removed.

Reply:
Until now the generator ignored only tests that had the `Ignore` attribute on the class, now I've applied the same rule also for methods.